vault/resource_pki_secret_backend_config_urls.go

```diff
@@ -84,6 +89,7 @@ func pkiSecretBackendConfigUrlsCreateUpdate(d *schema.ResourceData, meta interfa
 		"issuing_certificates":    d.Get("issuing_certificates"),
 		"crl_distribution_points": d.Get("crl_distribution_points"),
 		"ocsp_servers":            d.Get("ocsp_servers"),
+		"enable_templating":       d.Get("enable_templating"),
```

Comment:
Looks like this field was added in Vault 1.13. We should add a Vault Server version check in the resource and test files. You can see a examples of doing version checks in this PR: https://github.com/hashicorp/terraform-provider-vault/pull/1936/files#diff-3c100f49c020a28974d882dd5b34d276a7f825e3ccfaa18e584c1e26e2261d10R237

Reply:
Good catch, thanks! A new check was added for this for both create/update and read functions.
GetOk seems to always return false with a boolean, GetOkExists was used instead.

Reply:
GetOkExists is deprecated and we prefer to use `d.Get` for booleans. The SDK v2 that TFVP uses has some limitations around booleans and it is usually best to use `d.Get` for them.